src/main/java/net/atos/client/zgw/zrc/model/Geometry.java

```diff
@@ -32,4 +35,24 @@ public GeometryType getType() {
 
     @Override
     public abstract String toString();
+
+    @Override
+    public boolean equals(final Object o) {
+        if (this == o) {
+            return true;
+        }
+
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+
+        final Geometry geometry = (Geometry) o;
```

Comment:
Het stuk hierboven is in de afgeleidde class ook al gecontroleerd. Ik zou zeggen, hier niet meer doen dus. Eigenlijk zou ik zeggen hier helemaal geen equals implementeren want het type verschilt per afgeleidde class en die moet je sowieso al controleren. Dus er valt hier niets meer te controleren.